script/gen_requirements_all.py

```diff
@@ -24,7 +24,6 @@
     "decora",
     "decora_wifi",
     "envirophat",
-    "evdev",
```

Comment:
Do we know why this was here in the first place?

Reply:
evdev is linux only (keyboard interaction is super os specific). I assume this was added to prevent the keyboard_remote manifest from adding itself to requirements_all.txt (and requirements_test_all.txt). Which is making me realize this integration can never be added to tests?